### ISSUE TITLE
A couple more coins with auto exchange disabled on MPH

### ISSUE THREAD

**Issue by AcharyaOne** [Quake4/MindMiner#312]
Adzcoin
Auroracoin-Qubit
Ethersocial

**Comment by AcharyaOne** [comment]
You misunderstood... the following line at MPH.ps1,
$NoExchangeCoins = @("Bitcoin-Gold", "Bitcoin-Private", "Electroneum", "Geocoin", "Sexcoin", "Startcoin")
I pointed some coins with auto exchange disabled which aren't specified at MPH.ps1

**Comment by Quake4** [comment]
What are the coins?

**Comment by AcharyaOne** [comment]
Ethersocial uses Ethash so that's one to add at that line. Am I seeing it wrong?

**Comment by Quake4** [comment]
No. Ethersocial is no autoexchange coin and it detected automatically. This list for not right configured coins on MPH or for fast disable when autoexchage for coin closes.

**Comment by AcharyaOne** [comment]
My bad, I didn't know mindminer auto detected no exchange coins.

**Comment by Quake4** [comment]
forget about it.
return to subject.
is which autoexchange coins disabled in MM on MPH?

**Comment by AcharyaOne** [comment]
I pointed Adzcoin, Auroracoin-Qubit and Ethersocial which have auto exchange disabled on MPH which I thought it had to be specified in MM here:
NoExchangeCoins = @("Bitcoin-Gold", "Bitcoin-Private", "Electroneum", "Geocoin", "Sexcoin", "Startcoin")
These specified coins have auto exchange disabled on MPH and I thought the other 3 were missing here.

**Comment by AcharyaOne** [comment]
I think there was some miscommunication here. I initially pointed 3 coins which have auto exchange disabled on MPH and weren't specified on MM at NoExchangeCoins = @("Bitcoin-Gold", "Bitcoin-Private", "Electroneum", "Geocoin", "Sexcoin", "Startcoin")

You said that MM auto detects coins with auto exchange disabled and so it isn't needed to add those 3 coins in that line of MPH.ps1. Is this correct?

**Comment by Quake4** [comment]
Yes, possibly, now it six coins may delete from this list. But i do not check this.

**Comment by AcharyaOne** [comment]
MPH has been dead since they disabled auto exchange on bitcoin gold. I hardly see mining on MPH lately which is a shame.
